app.js

```diff
@@ -1,15 +1,25 @@
+const persist = require('choo-persist')
 const mount = require('choo/mount')
 const log = require('choo-log')
+const xtend = require('xtend')
 const choo = require('choo')
 
-const mainView = require('./pages/main')
+const opts = {
+  filter: (state) => {
+    state = xtend(state)
```

Comment:
7. We should get rid of `xtend` in the other places and then remove the dependency completely